vec/vec-common/src/main/java/com/foursoft/harness/vec/common/util/DateUtils.java

```diff
@@ -42,6 +42,22 @@
  */
 public final class DateUtils {
 
+    /**
+     * {@link Clock} which either uses UTC time
+     * or a fixed UTC time, controlled by defining {@code overwrite.clock}.
+     */
+    public static final Clock CLOCK;
+
+    static {
+        final String overwriteClock = System.getProperty("overwrite.clock", null);
+        if (overwriteClock != null) {
+            final Instant instant = Instant.parse(overwriteClock); // e.g. "2025-05-01T00:00:00Z"
```

Comment:
Please add a LOGGER.warn (slf4j) here that we override "now" with the provided value.